src/pres/loader/Filters.js

```diff
@@ -18,14 +18,17 @@ import * as SitePolicy from '../../app/SitePolicy'
 import { Radio, FormControlLabel, RadioGroup, FormHelperText, FormControl, FormLabel} from '@material-ui/core';
 import deepEqual from 'deep-equal'
 
+// Probably rename to Filters
 export default class User extends React.Component {
 
     constructor(props) {
         super(props)
+
+        let isAdvancedFiltersOpen = new URLSearchParams(window.location.search).get("advanced") === 'true'
```

Comment:
boolean param

also: it's arguably a bad thing that these param checks are spread across three different components. this was the simplest to start with, but i'm happy to change